buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/DistributedPerformanceTest.groovy

```diff
@@ -438,9 +461,23 @@ class DistributedPerformanceTest extends ReportGenerationPerformanceTest {
         }
     }
 
-    private static class ScenarioResult {
+    static class ScenarioResult {
         String name
         Map buildResult
         JUnitTestSuite testSuite
+
+        String getTestClassFullName() {
+            return testSuite.name
+        }
+
+        TestMethodResult toMethodResult(long counter) {
+            return new TestMethodResult(
+                ++counter,
```

Comment:
Increasing the counter here is pointless, since it is a method argument. You probably wanted to increase the counter where you call the method.